packages/webview_flutter/webview_flutter_android/example/android/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
+        classpath 'com.android.tools.build:gradle:4.1.3'
```

Comment:
is this change necessary to support the `onUrlChanged` feature?

The gradle version should be updated at some point to be in sync with the current Flutter stable (which uses version 4.1.0 at the moment). However it would be better to do so in a separate PR.

Reply:
I had not intended to update gradle, I think I just blindly accepted one of android studio's suggestions. Reverted it for now.